components/html-block/html-block.js

```diff
@@ -11,6 +11,12 @@ export const htmlBlockContentStyles = css`
 		font-weight: 400;
 		line-height: 1.2rem;
 	}
+	.d2l-html-block-no-vertical-margins > *:first-child {
```

Comment:
For a narrower net, I can restrict this to `h1-6`, `p`, `b`, `strong`, etc., though I'm not sure if that's necessary. Any inline styles on the rendered content will already take precedence here.

Reply:
This also doesn't work properly if the first (or last) node is a text node (or anything that isn't an element), because the child selectors ignore them. I'm not sure if there's a reasonable work-around for that, unless we apply this CSS dynamically based on what's in the rendered content, which... Well it seems sort of gross to me.

Reply:
What's the reason for including the universal selector, `*`?  I think this can be omitted.

Reply:
I originally had a more specific selector in there before I realized we would need it for every element, so I just replaced it with `*` to confirm. You're right that it's redundant though, so I'll remove it.